main.go

```diff
@@ -7,11 +7,17 @@ import (
 )
 
 func receiveAjax(w http.ResponseWriter, r *http.Request) {
+	const userErrorMsg = "Sorry, a server error occurred"
 	
 	fmt.Println("Received ajax data.")
 
 	// Invoke ParseForm before reading form values
-	r.ParseForm()
+	err := r.ParseForm()
```

Comment:
As much as feasible, errors from function calls should be handled or explicitly ignored.